internal/pkg/manifest/validate.go

```diff
@@ -850,16 +850,42 @@ func (a AdvancedCount) Validate() error {
 		}
 	}
 
+	// Validate combinations with cooldown
+	if !a.Cooldown.IsEmpty() {
+		if !a.hasScalingFieldsSet() {
+			return &errAtLeastOneFieldMustBeSpecified{
+				missingFields:    a.validScalingFields(),
+				conditionalField: "cooldown",
+			}
+		}
+		if !a.CPU.IsEmpty() {
+			if !a.CPU.ScalingConfig.IsEmpty() {
+				return &errFieldMutualExclusive{
+					firstField:  "cooldown",
+					secondField: "cpu_percentage/cooldown",
+				}
+			}
+		}
```

Comment:
Hmm I don't think they should be mutually exclusive with each other. The top level `cooldown` should be a general one where as any sub level ones could override its behavior.